docs/src/linux.md

```diff
@@ -0,0 +1,110 @@
+```sh
+mkdir -p ~/.local
+# extract zed to ~/.local/zed.app/
+tar -xvf <path/to/download>.tar.gz -C ~/.local
+# link the zed binary to ~/.local/bin (or another directory in your $PATH)
+ln -sf ~/.local/bin/zed ~/.local/zed.app/bin/zed
+```
+
+If you'd like integration with an XDG-compatible desktop environment, you will also need to install the `.desktop` file:
+
+```sh
+cp ~/.local/zed.app/share/applications/zed.desktop ~/.local/share/applications/zed.desktop
+sed -i "s|Icon=zed|Icon=$HOME/.local/zed.app/share/icons/hicolor/512x512/apps/zed.png|g" ~/.local/share/applications/zed.desktop
+sed -i "s|Exec=zed|Exec=$HOME/.local/zed.app/libexec/zed-editor|g" ~/.local/share/applications/zed.desktop
+```
+
+### Installing from source
+
+Zed is open source, and [you can install from source](./development/linux.md).
+
+### Installing via a package manager
+
+Several people have created third-party Zed packages for various linux package managers. You may be able to install zed using these packages.
+
+* [Alpine](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/zed)
+* [Nix](https://search.nixos.org/packages?show=zed-editor)
+* [Arch](https://archlinux.org/packages/extra/x86_64/zed/)
+* [Fedora](https://fedora.pkgs.org/40/terra/zed-preview-0:0.142.1-pre1.fc40.aarch64.rpm.html)
+* Please add others to this list!
+
+When installing a third-party package please be aware that it may not be completely up to date and may be slightly different from the Zed we package (a common change is to rename the binary to `zedit` to avoid conflicting with other packages).
+
+We'd love your help making Zed available for everyone. If Zed is not yet available for your package manager, and you would like to fix that, we have some notes on [how to do it](./development/linux.md#notes-for-packaging-zed).
+
+# Troubleshooting
+
+Linux works on a very large variety of systems configured in a multitude of different ways. We primarily test Zed on a vanilla Ubuntu setup, as it is the most common distribution our users use, that said we do expect it to work on a wide variety of machines.
+
+## Zed fails to start
+
+If you see an error like "/lib64/libc.so.6: version 'GLIBC_2.29' not found" it means that your distribution's version of glibc is too old. You can either upgrade your system, or [install Zed from source](./development/linux.md).
+
+## Zed fails to open windows
+
+Zed requires a GPU to run effectively. Under the hood we use [Vulkan](https://www.vulkan.org/) to communicate with your GPU. If you are seeing problems with performance, or Zed failing to load, it is possible that Vulkan is the culprit.
+
+A broken Vulkan results in error messages like: "Zed failed to open a window: NoSupportedDeviceFound" or "called `Result::unwrap()` on an `Err` value: ERROR_INITIALIZATION_FAILED".
+
+To begin troubleshooting vulkan, install the `vulkan-tools` package, and run:
+```sh
+$ vkcube
```

Comment:
We don't currently use sigils to indicate the shell prompt anywhere else in the docs:

```suggestion
vkcube
```